guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc

```diff
@@ -0,0 +1,149 @@
+[id="Updating-Disconnected-{project-context}-on-EL8_{context}"]
+= Updating a disconnected {ProjectServer} on {EL} 8
+
+You can update your disconnected {Project} on {RHEL} 8 by synchronizing the required repositories on the c
+onnected {Project} and syncing the content to the disconnected {Project} using `reposync`.
+
+.Procedure on the connected {ProjectServer}
+. Ensure that you have synchronized the following repositories in your connected {ProjectServer}:
++
+* {RepoRHEL8BaseOS}
+* {RepoRHEL8AppStream}
+* {RepoRHEL8ServerSatelliteServerProjectVersion}
+* {RepoRHEL8ServerSatelliteMaintenanceProjectVersion}
+. Download the debug certificate of the organization and store it locally at `/etc/pki/katello/certs/org-debug-cert.pem` or a location of your choosing.
+For more information, see {AdministeringDocURL}Creating_an_Organization_Debug_Certificate_admin[Creating an Organization Debug Certificate] in _{AdministeringDocTitle}_.
+. Create a Yum configuration file under `/etc/yum.repos.d`, such as `_{project-context}-disconnected_.repo`, with the following contents:
++
+[options="nowrap" subs="attributes"]
+----
+[{RepoRHEL8BaseOS}]
+name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/rhel8/8/x86_64/baseos/os
```

Comment:
I know this is copied from the old proc, but `_` for italics doesn't work if you don't do `subs="quotes"`. In other places we set `subs="+quotes,verbatim,attributes"` so probably should standardize on everywhere?